src/pages/Contents/Contents.scss

```
@@ -1 +1,300 @@
+      left: 350px;
+      right: 160px;
+      height: 236px;
+      padding: 20px 0 0 0;
+      h1 {
+        font-size: 33px;
+      }
+      div:nth-child(2) {
+        font-size: 17px;
+        color: rgba($color: gray, $alpha: 0.8);
+        margin-top: 10px;
+      }
+      div:nth-child(3) {
+        font-size: 17px;
+        margin: 17px 0;
+        padding: 10px 0;
+        border-top: $border;
+        border-bottom: $border;
+      }
+      .rating {
+        font-size: 12px;
+        opacity: 0.5;
+      }
+    }
+    @mixin addBtn {
+      width: 205px;
+      height: 40px;
+      border: none;
+      border-top-left-radius: 6px;
+      border-bottom-left-radius: 6px;
+      font-size: 17px;
+      font-weight: 600;
+      &:focus {
+        outline: none;
+      }
+    }
+    .buttons {
+      display: flex;
+    }
+    .addBtn_not_clicked {
+      @include addBtn;
+      color: white;
+      background-color: $themeColor;
+      &:hover {
+        .plus_icon {
+          transform: rotateZ(90deg);
+        }
+      }
+    }
+  }
+  .plus_icon {
+    transition: 0.2s ease-in-out;
+  }
+  .pinkFlag {
+    width: 20px;
+  }
+  .addBtn_clicked {
+    width: 205px;
+    height: 40px;
+    border: none;
+    border-top-left-radius: 6px;
+    border-bottom-left-radius: 6px;
+    font-size: 17px;
+    font-weight: 600;
+    &:focus {
+      outline: none;
+    }
+  }
+
+  .dropdownBtn_not_clicked {
+    margin-top: 22px;
+    width: 50px;
+    height: 40px;
+    color: white;
+    background-color: $themeColor;
+    border: none;
+    border-top-right-radius: 6px;
+    border-bottom-right-radius: 6px;
+    border-left: 1px solid rgb(231, 18, 82);
+    &:focus {
+      outline: none;
+    }
+    &:hover {
+      .dropdown_icon {
+        transform: translateY(2px);
+      }
+    }
+  }
+  .dropdown_icon {
+    transition: 0.2s ease-in-out;
+  }
+}
+.dropdownBtn_clicked {
+  margin-top: 22px;
+  width: 50px;
+  height: 40px;
+  border: none;
+  border-top-right-radius: 6px;
+  border-bottom-right-radius: 6px;
+  &:focus {
+    outline: none;
+  }
+}
+
+.dropdownBtn_not_clicked_color {
+  margin-top: 22px;
+  width: 50px;
+  height: 40px;
+  color: white;
+  background-color: $themeColor;
+  border: none;
+  border-left: 1px solid rgb(231, 18, 82);
+  &:focus {
+    outline: none;
+  }
+  &:hover {
+    .dropdown_icon {
+      transform: translateY(2px);
+    }
+  }
+}
+.dropdown_icon {
+  transition: 0.2s ease-in-out;
+}
+
+.dropdownBtn_clicked_color {
+  margin-top: 22px;
+  width: 50px;
+  height: 40px;
+  border: none;
+  &:focus {
+    outline: none;
+  }
+}
+
+main {
+  justify-content: center;
+  background-color: $grayColor;
+  display: flex;
+
+  article {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-right: 20px;
+    .comment_box {
+      display: flex;
+      justify-content: space-between;
+      width: 640px;
+      padding: 13px 20px;
+      @include box;
+      margin: 30px 0 -15px 0;
```

아래와 같이 작성해주실 수 있습니다. 

```suggestion
      margin: 30px 0 -15px;
```